demos/serverless/src/handlers.js

```diff
@@ -208,3 +208,105 @@ exports.end = async(event, context, callback) => {
   }).promise();
   callback(null, response);
 };
+
+exports.logs = async (event, context) => {
+
+  console.log(event);
+  var sequenceToken = "";
+  const logGroupName = "BrowserLogs";
+  var response = {
+    "statusCode": 200,
+    "headers": {},
+    "body": '',
+    "isBase64Encoded": false
+  };
+  try {
+    if (typeof JSON.parse(event.body) === 'undefined' || typeof JSON.parse(event.body).logs === 'undefined') {
+            response.body = "Empty";
+      return response;
+    } else {
+
+      var logs_arr = JSON.parse(event.body).logs;
+      //iterate through this array to get each jsons
+      var logSeqNo;
+      var timestamp;
+      var meetingID;
+      var attendeeID;
+      var logLevelType;
+      var appName;
+      var logMessage;
+
+
+      var cloudwatchlogs = new AWS.CloudWatchLogs({
+        apiVersion: '2014-03-28'
+      });
+      for (let i = 0; i < logs_arr.length; i++) {
+        logSeqNo = JSON.parse(logs_arr[i]).logSeqNo;
+        timestamp = JSON.parse(logs_arr[i]).timestamp;
+        meetingID = JSON.parse(logs_arr[i]).meetingID;
+        attendeeID = JSON.parse(logs_arr[i]).attendeeID;
+        logLevelType = JSON.parse(logs_arr[i]).logLevelType;
+        appName = JSON.parse(logs_arr[i]).appName;
+        logMessage = JSON.parse(logs_arr[i]).logMessage;
+
+        var msg = `${logSeqNo} AttendeeID: ${attendeeID} [${logLevelType}] ${logMessage}`;
+
+        var par = {
+          "logGroupName": logGroupName,
+          "logStreamNamePrefix": meetingID
+        }
+        if (sequenceToken != "") {
+          par = {
+            "logGroupName": logGroupName,
+            "logStreamNamePrefix": meetingID,
+            "nextToken": sequenceToken
```

Comment:
The sequenceToken you have defined in this handler is for `putLogEvents` api call. You cannot use it for `describeLogStreams` api.

Reply:
The `describeLogStreams` api will return its own sequence token which you should use incase there are multiple logstreams with same prefix